planner/core/plan_cost.go

```diff
@@ -88,13 +88,26 @@ func (p *PhysicalIndexLookUpReader) GetPlanCost(taskType property.TaskType) (flo
 	p.planCost = 0
 	// child's cost
 	for _, child := range []PhysicalPlan{p.indexPlan, p.tablePlan} {
-		childCost, err := child.GetPlanCost(taskType)
+		childCost, err := child.GetPlanCost(property.CopDoubleReadTaskType)
 		if err != nil {
 			return 0, err
 		}
 		p.planCost += childCost
 	}
 
+	// to keep compatible with the previous cost implementation, re-calculate table-scan cost by using index stats-count again (see copTask.finishIndexPlan).
```

Comment:
For example, in the case below:
```
mysql> explain select * from t use index(b) limit 10, 100;
+--------------------------------+---------+-----------+---------------------+--------------------------------------+
| id                             | estRows | task      | access object       | operator info                        |
+--------------------------------+---------+-----------+---------------------+--------------------------------------+
| IndexLookUp_12                 | 100.00  | root      |                     | limit embedded(offset:10, count:100) |
| ├─Limit_11(Build)              | 110.00  | cop[tikv] |                     | offset:0, count:110                  |
| │ └─IndexFullScan_9            | 110.00  | cop[tikv] | table:t, index:b(b) | keep order:false, stats:pseudo       |
| └─TableRowIDScan_10(Probe)     | 100.00  | cop[tikv] | table:t             | keep order:false, stats:pseudo       |
+--------------------------------+---------+-----------+---------------------+--------------------------------------+
```
To keep compatible with our previous implementation, we should use index stats-count `110` to calculate table-scan cost instead of using table stats-count `100`.